apps/auction2/components/auction_artworks/index.js

```diff
@@ -0,0 +1,35 @@
+import { default as React, PropTypes } from 'react';
+import { connect } from 'react-redux'
+import AuctionGridArtwork from '../auction_grid_artwork/index'
+import AuctionListArtwork from '../auction_list_artwork/index'
+
+function AuctionArtworks({ artworks, listView }) {
+  const DisplayComponent = listView ? AuctionListArtwork : AuctionGridArtwork
+  return (
+    <div className={'auction2-artworks'}>
+      {
+        artworks.map((artwork) => (
+          <DisplayComponent key={artwork.get('_id')} artwork={artwork} />
+        ))
+      }
+    </div>
+  )
+}
+
+AuctionArtworks.propTypes = {
+  artworks: PropTypes.array.isRequired,
+  listView: PropTypes.bool.isRequired
```

Comment:
I would suggest updating this to `isListView` to indicate that its a bool, and so that it reads more clearly in the UI code.